@xen-orchestra/build-template/src/index.spec.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-env jest */
+import buildTemplate from '.'
+
+it('builds a template', () => {
+  const replacer = buildTemplate('{name}_\\{name}_\\\\{name}_{id}_%_FOO', {
```

Comment:
Don't you want to test the `\` in another test?

Reply:
I don't think it's necessary. A test which test all combinations is enough. If one combination fail we'll know it.

Reply:
But it will be harder to pinpoint what's failing. But I guess they both make sense and I don't know what we usually do.

Reply:
So we'll know in the string which syntax is not transformed properly. Otherwise, we need to add a test for each syntax to know exactly which syntax fails.

Reply:
Yes, I think it would make more sense to test them one by one so that the Jest error would tell you exactly what's wrong when you break something. But I don't think we need to bother too much and testing it is already much better than not testing it, so I'm fine with that.